trust-center-agent/src/main/java/care/smith/fts/tca/rest/ConsentController.java

```diff
@@ -28,6 +30,7 @@
 import reactor.core.publisher.Mono;
 
 @Slf4j
+@Conditional(GicsPropertyNotEmptyCondition.class)
```

Comment:
Can we depend on `ConsentedPatientsProvider` here, rather then requiring gics to be configured?

Reply:
This conflicts with the `pageSize` coming from gicsConfiguration, obviously. We should fix this in another issue.